man/rgbasm.1

```diff
@@ -176,6 +177,13 @@ See the
 section for a list of warnings.
 .It Fl w
 Disable all warning output, even when turned into errors.
+.It Fl X Ar max_errors , Fl \-max-errors Ar max_errors
+If more than this number of errors (not warnings) occur, then abort the assembly process.
+.Fl X 0
+disables this behavior.
+The default is 100 if
+.Nm
+is running in a terminal, and 0 otherwise.
```

Comment:
```suggestion
If more than this number of errors (not warnings) occur, then abort the assembly process;
.Fl X 0
disables this behavior.
The default is 100 if
.Nm
is printing errors to a terminal, and 0 otherwise.
```